packages/test/end-to-end-tests/src/test/consensusOrderedCollectionEndToEndTests.spec.ts

```diff
@@ -52,8 +51,8 @@ function generate(
         afterEach(() => {
             args.reset();
         });
-        let container1: IContainer;
-        let container2: IContainer;
```

Comment:
it we make storageOnly an optional property on deltaManger can we get rid of this change?